crates/core/plugin_sm/src/plugin.rs

```diff
@@ -461,13 +468,54 @@ impl Plugin for ExternalPluginCommand {
         let command = self.command(LIST, None)?;
         let output = self.execute(command, logger).await?;
         if output.status.success() {
+            let filtered_output = match (&self.exclude, &self.include) {
+                (None, None) => output.stdout,
+                (Some(exclude), Some(include)) => {
+                    let exclude_filter = Regex::new(format!(r"^{}\t.*", exclude).as_str())?;
+                    let include_filter = Regex::new(format!(r"^{}\t.*", include).as_str())?;
+
+                    output
+                        .stdout
+                        .split_inclusive(|c| *c == b'\n')
+                        .map(|line| std::str::from_utf8(line).unwrap_or_default())
```

Comment:
todo: unwrap here will crash, should return error instead, unless this is intended

Reply:
I am dumb, this is `unwrap_or_default`, there is no problem.